src/mesh/installation/ecs.md

```diff
@@ -18,24 +18,22 @@ Kong Mesh supports tasks on the following launch types:
 
 The control plane itself also runs as an ECS service in the cluster.
 
-### Data plane tokens
+### Data plane authentication
 
 As part of joining and synchronizing with the mesh, every sidecar needs to authenticate with
-the control plane. On Kong Mesh, this is accomplished by using a data plane token.
-Typically on Universal mode, creating and managing data plane tokens is a manual step for the mesh operator.
-However, Kong Mesh on ECS handles automatically provisioning data plane tokens for your services.
+the control plane.
 
-An additional ECS token controller runs in the cluster with permissions to use
-the Kong Mesh API to create data plane tokens and put them in AWS secrets.
+With Kong Mesh, this is typically accomplished by using a data plane token.
```

Comment:
```suggestion
With {{site.mesh_product_name}}, this is typically accomplished by using a data plane token.
```

Reply:
I've opened another PR for this as it's unrelated to this change

Reply:
https://github.com/Kong/docs.konghq.com/pull/4721